tools/appveyor.psm1

```diff
@@ -326,9 +326,15 @@ function Invoke-AppVeyorTest
     Write-Host -Foreground Green 'Upload FullCLR test results'
     Update-AppVeyorTestResults -resultsFile $testResultsFileFullCLR
 
-    ## CodeCoverage
-    $env:CodeCoverageOutput = Split-Path -Parent (Get-PSOutput -Options (New-PSOptions -Configuration CodeCoverage))
- 
+    if(Test-DailyBuild)
+    {
+        ## Publish code coverage build, tests and OpenCover module to artifacts, so webhook has the information.
```

Comment:
Resolved. Using Push-AppVeyorArtifact instead.

---
In reply to: [89253329](https://github.com/PowerShell/PowerShell/pull/2760#discussion_r89253329) [](ancestors = 89253329)